### PR TITLE
support goToDefinition also for less

### PR DESCRIPTION
**Pull request by mariusGundersen**
This adds support for sourcemaps for less as well.

### CODE REVIEW

File: src/helpers/getCssExports.ts [hunk 1]
```diff
@@ -85,6 +86,9 @@ export const getCssExports = ({
               if (output === undefined) {
                 throw new Error('No Less output.');
               }
+
+              // eslint-disable-next-line @typescript-eslint/no-unnecessary-condition
```
Comment (mrmckeb) [inline src/helpers/getCssExports.ts:90]: Are the types wrong here? Did you encounter a situation where this could be falsey?

Comment (mariusGundersen) [reply src/helpers/getCssExports.ts:90]: No, I get an eslint error:
`Require statement not part of import statement. ` [@typescript-eslint/no-var-requires](https://typescript-eslint.io/rules/no-var-requires)

**Review by mrmckeb — APPROVED** [review]

**Comment by mrmckeb** [comment]
Thanks for this! I'm currently doing a big refactor, but will merge and release this by the end of the weekend.

**Comment by mrmckeb** [comment]
(I closed and reopened as the CI checks wouldn't run for some reason)

**Review by mrmckeb — APPROVED** [review]

**Comment by mrmckeb** [comment]
This has shipped in `v4.2.1`:
- https://github.com/mrmckeb/typescript-plugin-css-modules/releases/tag/v4.2.1
- https://github.com/mrmckeb/typescript-plugin-css-modules/releases/tag/v4.2.0

**Pull request merged**